python/cugraph/dask/opg_pagerank/pagerank.py

```diff
@@ -20,30 +20,54 @@
 from cugraph.opg.link_analysis import mg_pagerank_wrapper as mg_pagerank
 
 
-def common_func(sID, data, local_data):
-    print("Dataframe: ", data)
-    print("local data: ", local_data)
+def common_func(sID, data, local_data, alpha, max_iter, tol, personalization, nstart):
     sessionstate = worker_state(sID)
-    mg_pagerank.mg_pagerank(data[0], local_data, sessionstate['handle'])
-    return 1
+    return mg_pagerank.mg_pagerank(data[0],
+                                   local_data,
+                                   sessionstate['handle'],
+                                   alpha,
+                                   max_iter,
+                                   tol,
+                                   personalization,
+                                   nstart)
 
 
-def pagerank(input_graph):
-    print("INSIDE DASK PAGERANK")
+
+def pagerank(input_graph,
+             alpha=0.85,
+             personalization=None,
+             max_iter=100,
+             tol=1.0e-5,
+             nstart=None):
+    if tol != 1.0e-5:
+        raise Warning("Tolerance is currently not supported. Setting it to default 1.0e-5")
+    tol = 1.0e-5
+    if personalization is not None or nstart is not None:
+        raise Warning("personalization and nstart currently not supported. Setting them to None")
+    personalization = None
+    nstart = None
+
     client = default_client()
     _ddf = input_graph.edgelist.edgelist_df
     ddf = _ddf.sort_values(by='dst', ignore_index=True)
     data = DistributedDataHandler.create(data=ddf)
     comms = Comms(comms_p2p=False)
     comms.init(data.workers)
     local_data = data.calculate_local_data(comms)
```

Comment:
Let's review options for decoupling `calculate_local_data` from PageRank. From a user perspective, there is no way to separate ETL time than actual Pagerank time otherwise.

Reply:
For instance, we could have the option to call `addTransposedAdjList` before Pagerank. In this case, `calculate_local_data` would happen at that time, as well as coo2csr and renumbering potentially. Then Pagerank would only do this if CSC is not present already.

Just need to look at compatibility if we `addAdjList` or run OPG BFS afterward.